index.html

```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Document </title>
     <link rel="stylesheet" href="style.css"> 
 </head>
 <body>
-    <h1>Hello  Microverse </h1>
+    <h1>Hello Microverse </h1>
 
     
```

Comment:
## 2
### Your Summary
![SUMMARY](https://user-images.githubusercontent.com/93039359/184926052-b4d8164a-5ee0-493d-9009-36017656ded5.png)


### Descriptive Summary Example :heavy_check_mark:
![descriptive-summary](https://user-images.githubusercontent.com/93039359/184921477-5686f05d-45cf-4d90-8522-6db2f012e302.png)

- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: